### DISCUSSION TITLE
How to integrate and view the allure report on CI/CD tools like Jenkins and Github Action

### DISCUSSION THREAD

**Discussion by SonPhung95** [allure-framework/allure2#2816] [Questions & Support] [answered]
I need to view the Allure report after my scripts running on the CI/CD tool. Anyone gives some advices ? Thanks a lot in advance.

**Comment by baev** [comment]
Have you seen https://allurereport.org/docs/integrations/?

**Reply by SonPhung95** [reply]
yeah, I have read this. But how about the specific actions or steps to view the allure report after we build the scripts on Jenkins or Github Actions ?

**Reply by baev** [reply, marked as answer]
* Allure Jenkins Plugin — https://allurereport.org/docs/integrations-jenkins/
* Allure Github Actions — https://allurereport.org/docs/integrations-github/
